portfolio/src/main/webapp/style.css

```diff
@@ -5,8 +5,31 @@
   line-height: 1.1;
 }
 
-#fact-container {	
-  margin-top: 20px;	
+#fun-fact {
+    margin: 10px 0px;
```

Comment:
nitpick: style recommendation is 2 space indents for css. https://google.github.io/styleguide/htmlcssguide.html#Indentation

Reply:
In the cloud shell editor you can set the default tab size under: File > settings > open preferences > editor > tabSize

Reply:
Ah I didn't see that, thank you!